src/screens/EncounterDataScreen.tsx

```diff
@@ -98,14 +99,19 @@ export const EncounterDataScreen = ({ navigation }: StackScreenProps<any>) => {
                 verified={userData.verified}
                 placeholder={userData.imageUri}
               />
-              <TouchableOpacity onPress={selectImage} style={styles.editIconContainer}>
+              <TouchableOpacity
+                accessibilityLabel={`${a11yLabels.image} ${a11yLabels.button}`}
+                onPress={selectImage}
+                style={styles.editIconContainer}
+              >
                 <Icon.EditSetting color={colors.placeholder} />
               </TouchableOpacity>
             </WrapperRow>
           </Wrapper>
           <Wrapper style={styles.noPaddingTop}>
             <Label>{texts.encounter.firstName}</Label>
             <TextInput
+              accessibilityLabel={`${a11yLabels.firstName} ${a11yLabels.textInput}: ${firstName}`}
```

Comment:
will this be read out every time the input value changes?

Reply:
good question. if it does it is the same way for the `FormScreen`. I will check

Reply:
it does not read on every change.
it did however read "undefined" because it was not initialised with an empty string. so i fixed it ;)